src/command/init.rs

```diff
@@ -192,9 +192,25 @@ impl Init {
     }
 
     fn step_install_wasm_bindgen(&mut self, step: &Step, log: &Logger) -> Result<(), Error> {
-        info!(&log, "Installing wasm-bindgen-cli...");
-        bindgen::cargo_install_wasm_bindgen(step)?;
-        info!(&log, "Installing wasm-bindgen-cli was successful.");
+        let bindgen_version =
+            manifest::get_wasm_bindgen_version(&self.crate_path)?.ok_or(Error::CrateConfig {
+                message: "Unexpected error while parsing wasm-bindgen dependency version"
+                    .to_string(),
+            })?;
+
+        if bindgen_version.is_empty() {
+            let e = Error::CrateConfig {
```

Comment:
And here!

Reply:
then you can do `return Err(Error::crate_config("wasm-bindgen version dependency was empty"))`